plaidml/edsl/__init__.py

```diff
@@ -774,19 +774,30 @@ class SortDirection(enum.IntEnum):
     """Sort a tensor in descending order."""
 
 
-def argsort(x, axis=-1, direction=SortDirection.ASC):
+class SortStability(enum.IntEnum):
+    """Stability for a sort operation."""
+
+    STABLE = 0
+    """Sort a tensor in a stable way."""
```

Comment:
For clarity, perhaps instead give a definition of a stable sort, e.g. "Preserve the initial order of equivalent values".